src/main/frontend/handler/extract.cljs

```diff
@@ -36,7 +36,11 @@
                    (fn [[page blocks]]
                      (if page
                        (map (fn [block]
-                              (let [block-ref-pages (seq (:block/ref-pages block))]
+                              (let [block-ref-pages (seq (:block/ref-pages block))
+                                    is-root (= 0 (:block/level block))
+                                    block-ref-pages (if is-root
+                                                      (vec (:tags properties))
```

Comment:
The `tags` should already be included in the `:block/ref-pages`, check https://github.com/logseq/logseq/blob/master/src/main/frontend/format/block.cljs#L213.

But it might be a bug.